dts/bindings/serial/st,stm32-uart.yaml

```diff
@@ -16,3 +16,9 @@ properties:
       type: int
       description: Configures the parity of the adapter. Value 0 for none, 1 for odd and 2 for even parity.
       default: 0
+
+    dmas:
+      required: false
+
+    dma-names:
+      required: false
```

Comment:
You can just remove these as they are already included in base.yaml binding.
(sorry if this was not clear enough in the initial comment)